library/core/src/marker.rs

```diff
@@ -764,9 +757,9 @@ unsafe impl<T: ?Sized> Freeze for &mut T {}
 ///
 /// This trait is automatically implemented for almost every type.
 ///
-/// [`mem::replace`]: ../../std/mem/fn.replace.html
+/// [`mem::replace`]: crate::mem::replace
 /// [Pin]: crate::pin::Pin
```

Comment:
It looks like a few things were converted to intra-doc before!

Reply:
Thank matklad :) https://github.com/rust-lang/rust/pull/75780